test/README.md

```diff
@@ -0,0 +1,43 @@
+# libE57Format Testing
+
+Testing uses the [googletest](https://github.com/google/googletest) framework. The documentation for it may be [found here](https://google.github.io/googletest/).
```

Comment:
googletest is indeed one of the go-to solutions nowadays, widely used for the same purpose elsewhere.